backend/src/Entity/Appointment.php

```diff
@@ -21,6 +24,14 @@
             security: 'is_granted("ROLE_ADMIN")',
             normalizationContext: ['groups' => ['appointment-getall']]
         ),
+        //pass the establishment id as a parameter to the controller
+        new GetCollection(
+            uriTemplate: '/appointments/{id}/me',
```

Comment:
```suggestion
            uriTemplate: '/appointments/me',
```

et utilise un filter sur l'establishment id